remote-execution/Part 01 - The Basics of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
<p>Decentralized deep learning doesn't seem right to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dropoutlabs/tf-world-tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>